cpp/src/sort/stable_sort.cu

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-#include "sort_impl.cuh"
+#include <sort/sort_impl.cuh>
```

Comment:
Why was this include changed? I think the original version was correct.

Reply:
Specifically
> Use <> instead of "" unless the header is in the same directory as the source file.

from the [developer guide](https://docs.rapids.ai/api/libcudf/stable/developer_guide).